src/providers/user/user.tsx

```diff
@@ -65,6 +78,38 @@ export const ActiveUserProvider: React.FC = ({ children }) => {
     }
   )
 
+  useEffect(() => {
+    if (!isLoading) {
+      closeLongLoadingModal()
+      return
+    }
+    const timeout = setTimeout(() => {
+      openLongLoadingModal({
+        iconType: 'warning',
+        title: 'Something went wrong',
+        description: `Check out your Polkadot extension and allow app the access. If you can't do that, reload page and try again.`,
```

Comment:
This needs to be slightly tweaked but I'm out of words atm, will provide later

Reply:
Let's update the title: `Failed to connect with extension`
Let's update the description: `Seems you didn't enable the Polkadot extension and we cannot access your accounts. You can do that by clicking the extension icon in your browser toolbar. If you cannot do that, please reload the page and try again.`